kubernetes/metasploit/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+name: metasploit
+description: A Helm chart for deployiong metasploit framework into Kubernetes
+
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 0.1.0
+appVersion: 6.1.8
+keywords:
+  - metasploit
+  - security
+  - pentest
+  - penetration testing
+home: https://www.metasploit.com/
+icon: https://www.metasploit.com/includes/images/metasploit-r7-logo.svg
+sources:
+- https://github.com/rapid7/metasploit-framework
+maintainers:
+- email: gadi_naor@rapid7.com
+  name: Gadi Naor
```

Comment:
For the purposes of longevity it might make sense to point have the maintainers information set to the msf mailing list, or not have it at all:

https://github.com/rapid7/rex-socket/blob/12b5c5cefa390100a04defd8fb5d027581c5c6e1/rex-socket.gemspec#L9-L10